include/packager/live_packager.h

```diff
@@ -78,8 +78,8 @@ struct LiveConfig {
 
   OutputFormat format;
   TrackType track_type;
-  // TOOD: do we need non-integer durations?
-  double segment_duration_sec;
+  // TODO(david): do we need non-integer durations?
+  double segment_duration_sec = 5.0;
```

Comment:
As is, this value doesn't really do anything for live packaging.  I think it would be better remove it from the public interface and use a default value internally.

Reply:
Could you also remove this from live_packager_test.cc?  
The following `cmake` command will build live_packager_test.
```
cmake -DBUILD_SHARED_LIBS=1  -DBUILD_LIVE_TEST=1 ..
```

Reply:
Why did we add them under a flag? It is almost guaranteed we want them always built and visible by other tools

Reply:
Done